src/condition.cpp

```diff
@@ -1734,3 +1733,54 @@ bool ConditionSpellGroupCooldown::startCondition(Creature* creature)
 	}
 	return true;
 }
+
+bool ConditionDrunk::startCondition(Creature* creature)
+{
+	if (!Condition::startCondition(creature)) {
+		return false;
+	}
+
+	creature->setDrunkness(drunkness);
+	return true;
+}
+
+bool ConditionDrunk::updateCondition(const Condition* addCondition)
+{
+	const ConditionDrunk& conditionDrunk = static_cast<const ConditionDrunk&>(*addCondition);
+	return conditionDrunk.drunkness > drunkness;
+}
+
+void ConditionDrunk::addCondition(Creature* creature, const Condition* condition)
+{
+	const ConditionDrunk& conditionDrunk = static_cast<const ConditionDrunk&>(*condition);
```

Comment:
same, just use pointer?

Reply:
Question still present

Reply:
See bd2bb53. I squashed the commits and had to force an update. This snippet, indeed as GitHub points out, is outdated.